examples/inflate-helm-chart-crds/README.md

```diff
@@ -0,0 +1,65 @@
+# inflate-helm-chart: Remote Chart with CRDs
+
+### Overview
+
+This example demonstrates how to imperatively invoke the `inflate-helm-chart`
+function to inflate a helm chart that contains CRDs in the templated output.
```

Comment:
Question: May be I am too late to comment on this aspect. Did we consider using names other than `inflate` ? I think `render-helm-chart` would have been more intuitive for the end users.

Reply:
I chose `inflate-helm-chart` because that's what the other function was called and the kustomize builtin plugin is also called HelmChartInflationGenerator.

But also considering that the [description for `helm template`](https://helm.sh/docs/helm/helm_template/)  says "locally renders templates", I agree that `render-helm-chart` is more intuitive.

I will change it to `render-helm-chart` and confirm with @mengqiy before release.

Reply:
Great. I saw it has been renamed to `render-helm-chart`.